auditlog_security/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_auditlog_log_line_user,auditlog_log_line_user,model_auditlog_log_line,base.group_user,1,0,0,0
+access_auditlog_log_user,auditlog_log_user,model_auditlog_log,base.group_user,1,0,0,0
```

Comment:
This was from when we showed the rule and filtered line.
Standard auditlog still creates the"Logs" action, so with line 2, instead of an error, we see the read-only log, with ONLY the lines user can see, if any.
Looks better than an error on action execution IMHO.

Reply:
@gfcapalbo Yeaaaaa true but it could also give users access to something they shouldnt see. Shouldnt the real solution be to have the 'View Logs' button only visible to admins?